src/app/courses/view-courses/courses-view.component.html

```diff
@@ -4,7 +4,7 @@
 </mat-toolbar>
 <div class="space-container">
   <mat-toolbar class="primary-color font-size-1">
-    <span class="margin-lr-3"><h3>{{courseDetail.courseTitle}}</h3></span>
+    <span class="margin-lr-3, title">{{courseDetail.courseTitle}}</span>
```

Comment:
This is incorrect syntax for HTML classes.  They should be separated by spaces only.

Also please keep the `h3` tags.